openedx/features/idea/tests/views/test_idea_list_view.py

```diff
@@ -0,0 +1,20 @@
+from django.test import TestCase
+from django.urls import reverse
+from rest_framework import status
+
+from openedx.core.djangolib.testing.philu_utils import configure_philu_theme
+from openedx.features.idea.views import IdeaListingView
+
+
+class IdeaListingViewTest(TestCase):
+
+    @classmethod
+    def setUpClass(cls):
+        super(IdeaListingViewTest, cls).setUpClass()
+        configure_philu_theme()
+
+    def test_idea_list_view(self):
+        response = self.client.get(reverse('idea-listing'))
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
+        self.assertEqual(IdeaListingView.paginate_by, 9)
```

Comment:
Please let's assert it as we haven't customized pagination functionality of Django.

Reply:
Removed it bro